index.js

```diff
@@ -11,5 +11,5 @@ export async function setupPlugin() {
 
 // Capture event called heartbeat
 async function captureHeartbeat() {
-    await posthog.capture('heartbeat')
+    await posthog.capture('heartbeat', {timestamp:Date.now()})
```

Comment:
I'm not sure we even need that actually, given that library would just use now, otherwise maybe use a different property name

Reply:
```suggestion
    await posthog.capture('heartbeat', {timestamp_utc:Date.now()})
```